roles/network_plugin/calico/handlers/main.yml

```diff
@@ -16,5 +16,5 @@
   when: container_manager in ["docker"]
 
 - name: containerd | delete calico-node containers
-  shell: 'crictl pods --name calico-node-* -q | xargs -I% --no-run-if-empty bash -c "crictl stopp % && crictl rmp %"'
+  shell: 'crictl pods --name calico-node-* -q | xargs -I% --no-run-if-empty bash -c "crictl stopp % ; crictl rmp % ; true"'
```

Comment:
With this change there's a chance that containers will not be restarted silently. Why do you want it?